src/test/java/com/znsio/sample/e2e/screen/android/amazon/AmazonProductViewAndroid.java

```diff
@@ -0,0 +1,66 @@
+package com.znsio.sample.e2e.screen.android.amazon;
+
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonProductViewScreen;
+import com.znsio.sample.e2e.screen.web.amazon.AmazonHomeScreenWeb;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+
+import static com.znsio.e2e.tools.Wait.waitFor;
+
+public class AmazonProductViewAndroid extends AmazonProductViewScreen{
+
+    private final Driver driver;
+    private final Visual visually;
+    private static final String SCREEN_NAME = AmazonHomeScreenWeb.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+
+    private static By firstProduct = By.xpath("//android.view.View[contains(@content-desc, \""+ SAMPLE_TEST_CONTEXT.PRODUCT_NAME+"\")]/android.view.View[2]");
+    private static final By productTitle = By.xpath("//android.view.View[@resource-id = \"title\"]");
+    private static By addCartButton = By.xpath("//android.widget.Button[@resource-id = \"add-to-cart-button\"]");
+    private static final By addedToCartMsg = By.xpath("//div[@id='attachDisplayAddBaseAlert']/span");
+
```

Comment:
xpath variables naming convention is not correct.

Reply:
corrected